README.md

```diff
@@ -1,35 +1,42 @@
-# jsonlogic-swift
-
-[![CI Status](http://img.shields.io/travis/advantagefse/json-logic-swift.svg?style=flat)](https://travis-ci.org/advantagefse/json-logic-swift)
-[![Version](https://img.shields.io/cocoapods/v/jsonlogic.svg?style=flat)](https://cocoapods.org/pods/jsonlogic)
-[![Platform](https://img.shields.io/cocoapods/p/jsonlogic.svg?style=flat)](https://cocoapods.org/pods/jsonlogic)
-[![codecov](https://codecov.io/gh/advantagefse/json-logic-swift/branch/master/graph/badge.svg)](https://codecov.io/gh/advantagefse/json-logic-swift)
-
-A native Swift JsonLogic implementation. This parser accepts [JsonLogic](http://jsonlogic.com) 
-rules and executes them. 
-
-JsonLogic is a way to write rules that involve computations in JSON 
+<p align="center">
+ <a href="https://www.coronawarn.app/en/"><img src="https://raw.githubusercontent.com/corona-warn-app/cwa-documentation/master/images/CWA_title.png" width="400"></a>
+</p>
+
+<hr />
+
+<p align="center">
+    <a href="https://api.reuse.software/info/github.com/corona-warn-app/json-functions-swift" title="REUSE Status">
+      <img src="https://api.reuse.software/badge/github.com/corona-warn-app/json-functions-swift">
+    </a>
+</p>
+<p align="center">
+    <a href="#usage">Usage Guide</a> •
+    <a href="#requirements">Requirements</a> •
+    <a href="#documentation">Documentation</a> •
+    <a href="#support-and-feedback">Support and Feedback</a> •
+    <a href="#how-to-contribute">How to contribute</a> •
+    <a href="#licensing">Licensing</a> •
+    <a href="https://www.coronawarn.app/en/">Web Site</a>
+</p>
+<hr />
+
+# Corona-Warn-App: jsonlogic-swift
+
+A native Swift JsonLogic implementation. This parser accepts [JsonLogic](http://jsonlogic.com)
+rules and executes them.
+
+JsonLogic is a way to write rules that involve computations in JSON
 format, these can be applied on JSON data with consistent results. So you can share between server and clients rules in a common format. Original JS JsonLogic implementation is developed by Jeremy Wadhams.
```

Comment:
Should we update this to say that this is a JsonFunctions implementation and not just JsonLogic? Maybe that it is backwards compatible with JsonLogic?